awscli/customizations/awslambda.py

```diff
@@ -38,9 +43,7 @@ def validate_is_zip_file(cli_argument, value, **kwargs):
 
 def _extract_code_and_zip_file_arguments(session, argument_table, **kwargs):
     argument_table['zip-file'] = ZipFileArgument(
-        'zip-file', help_text=('The path to the zip file of the code you '
-                               'are uploading. Example: fileb://code.zip'),
-        cli_type_name='blob')
+        'zip-file', help_text=ZIP_DOCSTRING, cli_type_name='blob')
```

Comment:
No, it just makes it into a newline.
![screen shot 2016-01-11 at 1 43 38 pm](https://cloud.githubusercontent.com/assets/2643092/12247551/5f230f56-b869-11e5-8dc4-d59ad047bb44.png)